### PR TITLE
Customizable timestamps format

### PR DESCRIPTION
**Pull request by ctaoist**
Added `with_custom_timestamps` to customize timestamps format.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -111,6 +113,7 @@ impl SimpleLogger {
 
             #[cfg(feature = "timestamps")]
             timestamps: Timestamps::Utc,
+            timeformat: TIMESTAMP_FORMAT_OFFSET,
```
Comment (borntyping) [inline src/lib.rs:116]:
```suggestion
            #[cfg(feature = "timestamps")]
            timeformat: TIMESTAMP_FORMAT_OFFSET,
```

**Review by borntyping — COMMENTED** [review]
Very neat and tidy. Any chance you could add an example in the `examples/` directory?

**Comment by ctaoist** [comment]
Added `examples/timestamps_format.rs` and could also customize format for utc timestamps now.

**Comment by borntyping** [comment]
Neat! I'll try and get this merged and released soon.

**Comment by borntyping** [comment]
Released as [4.1.0](https://github.com/borntyping/rust-simple_logger/releases/tag/v4.1.0) with some minor changes - the only breaking one would be that `with_custom_timestamps` was renamed to `with_timestamp_format`. Thank you for contributing!

**Pull request merged**